interface/forms/procedure_order/new.php

```diff
@@ -382,16 +382,33 @@ function addProcLine(flag = false) {
     let i = 0;
```

Comment:
Padding is still wrong in this file
![image](https://user-images.githubusercontent.com/46497068/91352733-3ce71f80-e7b8-11ea-9f30-4b71bf53d5b3.png)

Reply:
@stu01509 Still wrong